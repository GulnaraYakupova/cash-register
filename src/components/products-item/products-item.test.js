import React from 'react';
import ProductsItem from './products-item';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

const data = {
  name: 'Гречка',
  amount: 2,
  price: 150,
}

it ('Should render ProductsItem component', () => {
  const component = shallow(
  <ProductsItem
    itemData={data}
    onProductsItemClick={()=>{}}
   />
  );
  expect(component).toMatchSnapshot();
});