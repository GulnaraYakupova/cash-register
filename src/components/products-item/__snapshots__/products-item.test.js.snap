// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ProductsItem component 1`] = `ShallowWrapper {}`;
